### PR TITLE
[tarantool] initial integration

### PR DESCRIPTION
**Pull request by ligurio**
How-To Build: https://www.tarantool.io/en/doc/latest/dev_guide/building_from_source/

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
Who are some of the users of tarantool?

**Comment by ligurio** [comment]
> Who are some of the users of tarantool?

Avito, Mastercard, Badoo, Yota etc. See https://www.tarantool.io/en/

**Comment by ligurio** [comment]
@jonathanmetzman I'm close to finish integration (waiting for merge patch to the project). Will it be possible to merge?

**Comment by jonathanmetzman** [comment]
> @jonathanmetzman I'm close to finish integration (waiting for merge patch to the project). Will it be possible to merge?

Yes I will merge this if upstream wants the OSS-Fuzz integration and once CI is green.

**Comment by ligurio** [comment]
@jonathanmetzman I'm ready to review

File: projects/tarantool/build.sh
Comment (jonathanmetzman) [inline projects/tarantool/build.sh:3]: Can you remove this line please?

**Review by jonathanmetzman — CHANGES_REQUESTED** [review]
LGTM with one nit.
It's just easier for us if you use the standard copyright lines as everyone else.

File: projects/tarantool/build.sh
Comment (ligurio) [reply projects/tarantool/build.sh:3]: Sure, removed.

BTW, did you know that a number of projects have additional copyrights?
Just run `grep -R Copyright projects/ | grep -v Google` in a root directory.

Comment (jonathanmetzman) [reply projects/tarantool/build.sh]: Thanks.
I didn't know this. Thanks for pointing it out.

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Pull request merged**
